### PR TITLE
release: Update image tags for v0.11.1 (#440)

### PR DESCRIPTION
**Pull request by heyselbi**
Update version tags in the following files:
- config/default/config-defaults.yaml
- config/dependencies/quickstart.yaml
- config/manager/kustomization.yaml
- docs/component-versions.md
- docs/install/install-script.md
- docs/quickstart.md
- scripts/setup_user_namespaces.sh

Update branch rule pattern in .github/workflows/*.yml to run workflows on release-0.11.1 branch

---------

#### PR checklist

Checklist items below are applicable for development targeted to both fast and stable branches/tags
- [x] Unit tests pass locally
- [ ] FVT tests pass locally
- [ ] If the PR adds a new container image or updates the tag of an existing image (not build within cpaas), is the corresponding change made in live-builder and cpaas-midstream to add/update the image tag in the operator CSV? Link the PRs if applicable

Checklist items below are applicable for development targeted to both fast and stable branches/tags
- [ ] Tested modelmesh serving deployment with odh-manifests and ran odh-manifests-e2e tests locally

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -3,7 +3,9 @@ name: Build
 on:
   workflow_dispatch:
   push:
-    branches: [master]
+    branches:
+      - main
+      - 'release-0.11.1'
```
Comment (Jooho) [inline .github/workflows/build.yml:8]: please roll back this part. we don't use gitaction for build

File: .github/workflows/fvt.yml [hunk 2]
```diff
@@ -3,7 +3,9 @@ name: FVT
 on:
   workflow_dispatch:
   pull_request:
-    branches: [master]
+    branches:
+      - main
+      - 'release-0.11.1'
```
Comment (Jooho) [inline .github/workflows/fvt.yml:8]: please roll back this part. we don't use gitaction for fvt

File: main.go [hunk 3]
```diff
@@ -304,7 +304,7 @@ func main() {
 			for key, value := range etcdSecret.Data {
 				if key == modelmesh.EtcdSecretKey {
 					var data map[string]json.RawMessage
-					err := json.Unmarshal(value, &data)
+					err = json.Unmarshal(value, &data)
```
Comment (danielezonca) [inline main.go:307]: Easter egg? :)

Comment (heyselbi) [reply main.go:307]: @danielezonca lol upstream lint check failed and I had to address it. But then we removed upstream lint check xD but I still kept the change.

**Pull request merged**